pizza.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="de">
```

Comment:
Die Sprache sollte in allen dokumenten auf "en" geändert werden.

Reply:
Stimmt!!